README.md

```diff
@@ -40,7 +40,7 @@ The Modules above explained
 Description of utilising Spring Boot
 
 1) [Running the Example](#running-the-example) -
```

Comment:
```suggestion
1) [Running the Examples](#running-the-examples) -
```

Reply:
Sorry, you missed this one Alex - I suggested the title of the link to be plural too